clients/java-logger/grpc-test-app/src/main/java/abcxyz/lumberjack/test/talker/TalkerService.java

```diff
@@ -112,20 +114,22 @@ private void blockUntilShutdown() throws InterruptedException {
   }
 
   static class JWKHandler implements HttpHandler {
-    // Matching private key here:
-    // https://github.com/abcxyz/lumberjack/blob/main/integration/testrunner/grpcrunner/grpc.go#L59
-    private static final String PUBLIC_JWK =
-        "{"
-            + "\"crv\": \"P-256\","
-            + "\"kid\": \"integ-key\","
-            + "\"kty\": \"EC\","
-            + "\"x\": \"hBWj8vw5LkPRWbCr45k0cOarIcWgApM03mSYF911de4\","
-            + "\"y\": \"atcBji-0fTfKQu46NsW0votcBrDIs_gFp4YWSEHDUyo\""
-            + "}";
+    private static byte[] parsePublicKey() throws Exception {
+      byte[] data = Files.readAllBytes(Paths.get("test_public_key.key"));
+      return data;
+    }
 
     @Override
     public void handle(HttpExchange t) throws IOException {
-      String response = String.format("{\"keys\": [%s]}", PUBLIC_JWK);
+      byte[] publicKey;
+      try {
+        publicKey = parsePublicKey();
```

Comment:
Same here and in the http test app, should we remove this parsePublicKey and directly read the files here?